### PR TITLE
(CDAP-1741) Docs for app logbacks

### PR DESCRIPTION
**Pull request by ParthGandhi**
Mostly cherry-picked from https://github.com/caskdata/cdap/pull/1607.

The only change is to `installation.rst` with an additional instruction for upgrade.

### CODE REVIEW

**Comment by awholegunch** [comment]
LGTM.

**Comment by ParthGandhi** [comment]
I'll wait for @poornachandra to review the upgrade instructions before merging.

**Comment by awholegunch** [comment]
Good plan.

**Comment by poornachandra** [comment]
LGTM

File: cdap-docs/admin-manual/source/operations/logback.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+====================================
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/operations/logback.rst]: add a reference before the over line:

```
.. _master-service-logging:
```

File: cdap-docs/reference-manual/source/glossary.rst [hunk 2]
```diff
@@ -98,5 +98,16 @@ Glossary
    Apache Hadoop
       See :term:`Hadoop`.
 
+   Master Services
+      CDAP system services that are run in YARN containers like Transaction Service,
+      Dataset Executor, Log Saver, Metrics Processor, etc.
+
+
+.. |(TM)| unicode:: U+2122 .. trademark sign
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/glossary.rst]: Don't add these two replaces, as I have already merged them in as part of the common conf.py used to build the docs.

**Comment by awholegunch** [comment]
One minor comment, otherwise LGTM. Fix and merge!

**Pull request merged**